form/src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import Question from "./Question";
+import Result from "./Result";
+
+const Form = () => {
+  const [answer, setAnswer] = useState(" ");
+  const [isSubmit, setIsSubmit] = useState(false);
+  const handleCheckInput = (e) => {
+    setAnswer(e.target.value);
+  };
+  const handleClick = (e) => {
```

Comment:
choose better and more understandable function names. avoid generic names

Reply:
I am still not sure about how `handleCheckInput` is meaningful name. It doesn't check anything at all.